src/runner.ts

```diff
@@ -1,17 +1,163 @@
 import * as core from "@actions/core";
 import * as types from './types';
+import * as cp from 'child_process';
+import * as os from 'os';
+import * as fs from 'fs';
+import * as pt from 'path';
+import * as glob from 'glob';
+import {messages, messagesFormatter} from './messages';
+
+export interface RunOptions {
+    /* Specify a path to folder or a file for the Parasoft coverage report */
+    report: string;
```

Comment:
What is the behavior on other platforms we support? Should we instead allow users to specify something like a minimatch pattern to locate reports, rather than distinguishing between a directory and a report?
Also, I may have additional comments after our discussion.